src/Compilers/CSharp/Portable/Symbols/NullableAnnotation.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// The nullable annotations that can apply in source.
```

Comment:
Consider adding info in the comment on how this works with types nested in a generic.